daemon/graphdriver/lcow/lcow.go

```diff
@@ -140,8 +140,8 @@ type layerDetails struct {
 
 // deletefiles is a helper function for initialisation where we delete any
 // left-over scratch files in case we were previously forcibly terminated.
-func deletefiles(path string, f os.FileInfo, err error) error {
-	if strings.HasSuffix(f.Name(), ".vhdx") {
+func deletefiles(path string, f os.FileInfo, _ error) error {
+	if f != nil && strings.HasSuffix(f.Name(), ".vhdx") {
```

Comment:
Is it correct in here that the error is completely unhandled?

Reply:
Not sure TBH. I initially changed it to return on error, but I couldn't find if it was done for a reason (and not familiar enough with the LCOW code). Given that we're phasing out LCOW, I decided to keep the behavior to what it was, but happy to change if you think that's better